docs/titan-basics/workspace-windows.md

```diff
@@ -108,9 +110,11 @@ use the \[Close All\] softkey.
 ### Context Menu Buttons
 
 On all consoles apart from Sapphire Touch and Titan Go, **context menu 
-buttons** are automatically shown below the prompt area on the top right 
-of the main screen. These buttons contain useful functions for the current
+buttons** appear which contain useful functions for the current
 active window and change depending on the window. \
+On Diamond 9 the context menu buttons are in the top left hand corner of the right
+hand screen - on all other consoles they are below the prompt area on the top right 
```

Comment:
I'm not sure this is true... titan go and sapphire touch both don't have the context buttons on screen, and are only visible in the context menu of a window.

Reply:
It does say that in the preceding and following paragraphs